install_config/aggregate_logging.adoc

```diff
@@ -102,308 +88,409 @@ deployed. To control component placement, specify node selectors per component t
+these instances on nodes reserved or optimized for running them.
+For example, the selector could be `node-type=infrastructure`. At least
+one active node must have this label before Elasticsearch will deploy.
+
+|`openshift_logging_kibana_ops_nodeselector`
+|A node selector that specifies which nodes are eligible targets
+for deploying Kibana instances.
+
+|`openshift_logging_curator_ops_nodeselector`
+|A node selector that specifies which nodes are eligible targets
+for deploying Curator instances.
 |===
 
-Running the deployer creates a deployer pod and prints its name. Wait until the
-pod is running. This can take up to a few minutes for {product-title} to
-retrieve the deployer image from the registry. Watch its process with:
+//. Create a xref:../dev_guide/secrets.adoc#dev-guide-secrets[secret] to provide
+//security-related files to the deployer. Providing the secret is optional, and
+//the objects will be randomly generated if not supplied.
+//+
+//You can supply the following files when creating a new secret, for example:
+//+
+//----
+//$ oc create secret generic logging-deployer \
+//    --from-file kibana.crt=/path/to/cert \
+//    --from-file kibana.key=/path/to/key
+//----
+//
+//[cols="3,7",options="header"]
+//|===
+//|File Name
+//|Description
+//
+//|*_openshift_logging_kibana_cert_*
+//|A browser-facing certificate for the Kibana server.
+//
+//|*_openshift_logging_kibana_key_*
+//|A key to be used with the Kibana certificate.
+//
+//|*_openshift_logging_kibana_ops_cert_*
+//|A browser-facing certificate for the Ops Kibana server.
+//
+//|*_openshift_logging_kibana_ops_key_*
+//|A key to be used with the Ops Kibana certificate.
+//
+//|*_openshift_logging_kibana_ops_ca_*
+//|The absolute path on the control node to the CA file to use
+//for the public facing ops kibana certs.
+//
+//|*_server-tls.json_*
+//|JSON TLS options to override the Kibana server defaults. Refer to
+//https://nodejs.org/api/tls.html#tls_tls_connect_options_callback[Node.JS] docs
+//for available options.
+//
+//|*_ca.crt_*
+//|A certificate for a CA that will be used to sign all certificates generated by
+//the deployer.
+//
+//|*_ca.key_*
+//|A matching CA key.
+//|===
+
+[[deploying-the-efk-stack]]
+== Deploying the EFK Stack
+
+The EFK stack is deployed using an Ansible playbook to
+the the EFK components. Run the playbook from the default Openshift Ansible location
+using the default
+xref:../install_config/install/advanced_install.adoc[inventory] file.
 
 ----
-$ oc get pod/<pod_name> -w
+$ ansible-playbook playbooks/common/openshift-cluster/openshift_logging.yml
 ----
 
-It will eventually enter *Running* status and end in *Complete* status. If takes
-too long to start, retrieve more details about the pod and any associated events
-with:
+Running the playbook deploys all the resources such as Secrets, ServiceAccounts, and
+DeploymentConfigs to support the stack. It will wait until the component pods
```

Comment:
I'm not too sure what the "It will wait..." sentence is referring to. Would this be accurate?

"Running the playbook deploys all resources needed to support the stack; such as Secrets, ServiceAccounts, and DeploymentConfigs. The playbook waits to deploy the component pods until the the stack is running. If the steps fail..."